Yosemite/Yosemite/Model/Model.swift

```diff
@@ -98,6 +98,7 @@ public typealias StoredProductSettings = Networking.StoredProductSettings
 public typealias CardReader = Hardware.CardReader
 public typealias CardReaderEvent = Hardware.CardReaderEvent
 public typealias CardReaderServiceDiscoveryStatus = Hardware.CardReaderServiceDiscoveryStatus
+public typealias CardReaderServiceError = Hardware.CardReaderServiceError
```

Comment:
This was going to be necessary eventually, so that the UI can switch on the errors emitted by the card reader, so we might as well do it now.